pangea/contrib/covid19/tasks.py

```diff
@@ -28,10 +29,26 @@ def process_covid19(user_id, reads_path):
         results_object_name = f'covid19/results/{result_filepath}'
         upload_file(result_filepath, object_name=results_object_name)
 
+        user = PangeaUser.objects.get(pk=user_id)
+        grp = user.personal_org.core_sample_group
+        sample = grp.create_sample(
+            name=reads_path.split('/')[-1].split('.fq')[0].split('.fastq')[0]
+        )
+        sample.save()
+        ar = sample.create_analysis_result(
+            module_name='covid19_kraken2'
+        )
+        ar.save()
+        field = ar.create_field(name='report', stored_data={
+            '__type__': 's3',
+            'endpoint_url': settings.S3_ENDPOINT,
+            'uri': f's3://{results_object_name}', 
```

Comment:
There will be however I'm pretty sure that's not a valid S3 URI. S3 URI's don't specify the endpoint URL because amazon thought it would always be theirs